server/utils.py

```diff
@@ -22,11 +22,13 @@ def iter_get(client):
 next_re = re.compile(r'<(.*)>; rel="next"')
 def _get_next_url(resp):
     links = resp.headers.get('link', '')
-    next_match = next_re.match(links)
-    if next_match:
-        return next_match.groups()[0]
-    else:
-        return None
+    split_links = links.split(",")
+    for item in split_links:
+        next_match = next_re.match(item)
+        if next_match:
+            return next_match.groups()[0]
+        else:
+            return None
```

Comment:
I'm confused about this part – won't you always only get through one iteration of the for loop in this case? What purpose is the for loop serving in that case?

Reply:
sure, good question. split_links list may contain one or more links (prev, next, last, first). The for loop iterates through each item on the list so the regex can be applied for each list element. 
If the regex finds a match, it returns the matching link, otherwise it assumes that it has reached the last page and returns none.